client/src/Components/Modal/EmbedModal.jsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+
+import styles from './EmbedModal.css';
+
+const EmbedModal = ({showEmbedModal}) => {
+
+  return (
+    <div className={styles.modal}>
+      <section className={styles.modal_main}>
+        <div className={styles.form_heading}>
+          <h1 className={styles.heading}>Embed This Review</h1>
+          <p className={styles.close} onClick={() => showEmbedModal()}>X</p>
+        </div>
+
+        <div className={styles.tagline}>
+          <h4>Embedding Code</h4>
+          <span className={styles.tagline_text}>Add this review to your website by copying the code below</span>
+        </div>
+
+        <div className={styles.restaurant_url}>
+          <input className={styles.site_url} type="text" placeholder='<span class="yelp-review" data-review-id="BEKynAKsr2J-kSPscp36qQ" data-hostname="www.yelp.com">Read <a href="https://www.yelp.com/user_details?userid=Alm1ibQS3RqzC5LCInDhGw"'></input>
+        </div>
+
+        <div className={styles.copy_btn}>Copy Code</div>
+
+        <div className={styles.preview}>
+          <h4 className={styles.heading}>Preview</h4>
+          <div className={styles.preview_container}>
+            <div className={styles.preview_detail}>
+              <div className={styles.restaurant_preview__heading}>
+                <h4 className={styles.restaurant_title}>Taco Boy</h4>
+                <div className={styles.preview_rating}>
+                  <div className={styles.stars}>
+                    <span className={styles.review_stars}> <i className="fa fa-star" aria-hidden="true"></i> </span>
+                    <span className={styles.review_stars}> <i className="fa fa-star" aria-hidden="true"></i> </span>
+                    <span className={styles.review_stars}> <i className="fa fa-star" aria-hidden="true"></i> </span>
+                    <span className={styles.review_stars}> <i className="fa fa-star" aria-hidden="true"></i> </span>
+                    <span className={styles.review_stars_grey}> <i className="fa fa-star" aria-hidden="true"></i> </span>
+                    <span className="review_count">9 reviews</span>
```

Comment:
Same comment as above for review count, check throughout too, just wondering it it's intentional